src/packages/name-service/utils.ts

```diff
@@ -39,11 +39,12 @@ export const fetchNameDetails = async (blockstackId: string, bnsNodes: string[])
     return response;
 };
 
-const bnsResolveName = async (baseUrl: string, blockstackId: string): Promise<object> => {
+const bnsResolveName = async (baseUrl: string, blockstackId: string, tag?: string): Promise<object> => {
     const options = {
         baseUrl,
         json: true,
         method: "GET",
+        qs: tag,
```

Comment:
qs needs to be an object - so you will need to convert 'tag' which is a string, to an object - 
`qs: {tag: tag}`

Reply:
Handle question mark case here